model/reliability/query.go

```diff
@@ -0,0 +1,16 @@
+package reliability
+
+import (
+	"github.com/mongodb/grip"
+)
+
+// ValidateForTaskReliability validates that the StartAt struct is valid for use with test stats.
+func (f *TaskReliabilityFilter) ValidateForTaskReliability() error {
+	catcher := grip.NewBasicCatcher()
```

Comment:
Will this be implemented later? If so I would just return errors.New("unimplemented")

Reply:
this returns nil, I'd avoid putting boilerplate here until you need it

Reply:
ok.